SPECS/python-ssh-python/python-ssh-python.spec

```diff
@@ -0,0 +1,100 @@
+%global modname ssh-python
+Name:           python-%{modname}
+Version:        0.9.0
+Release:        3%{?dist}
+Summary:        Bindings for libssh C library
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/ParallelSSH/ssh-python
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{modname}-%{version}.tar.gz
+Patch0001:      0001-Set-master_doc-to-index-in-conf.py-for-sphinx.patch
+
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  libssh-devel
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-Cython
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+
+%if %{with_check}
+BuildRequires:  python3-atomicwrites
+BuildRequires:  python3-attrs
+BuildRequires:  python3-docutils
+BuildRequires:  python3-pluggy
+BuildRequires:  python3-pygments
+BuildRequires:  python3-six
+BuildRequires:  python3-pip
+BuildRequires:  python3-pytest
```

Comment:
We can install these in the check section using pip, instead of requiring them here.

Reply:
Done, using pip to install the requirements.